Cargo.toml

```diff
@@ -22,17 +22,17 @@ name = "yagna"
 path = "core/serv/src/main.rs"
 
 [dependencies]
-ya-activity = "0.2"
+ya-activity = "0.3"
 ya-compile-time-utils = "0.1"
-ya-dummy-driver = { version = "0.1", optional = true }
+ya-dummy-driver = { version = "0.2", optional = true }
 ya-file-logging = "0.1"
-ya-gnt-driver = { version = "0.1", optional = true }
-ya-zksync-driver = { version = "0.1", optional = true }
+ya-gnt-driver = { version = "0.2", optional = true }
+ya-zksync-driver = { version = "0.2", optional = true }
 ya-identity = "0.2"
-ya-market = "0.2"
+ya-market = "0.3"
 ya-metrics = "0.1"
-ya-net = { version = "0.1", features = ["service"] }
-ya-payment = "0.1"
+ya-net = { version = "0.2", features = ["service"] }
+ya-payment = "0.3"
```

Comment:
@maaktweluit pls revert to "0.2" as I've already bumped it from "0.1"

Reply:
Whoops, missed that.. Reverted back to 0.2 :)

Reply:
thx